deap/gp.py

```diff
@@ -1045,7 +1058,8 @@ def halflifefunc(x):
     record = stats.compile(population) if stats else {}
     logbook.record(gen=0, nevals=len(invalid_ind), **record)
     if verbose:
-        print logbook.stream
+        print
```

Comment:
This change looks weird. Please revert back to
```python
print logbook.stream
```

Reply:
Sorry, pycharm's python3 auto-indentation :(